netdata.spec.in

```diff
@@ -132,12 +134,14 @@ rm -rf $RPM_BUILD_ROOT
 %config(noreplace) %{_sysconfdir}/%{name}/*.conf
 %config(noreplace) %{_sysconfdir}/%{name}/python.d/*.conf
 %dir %{_sysconfdir}/%{name}
-%{?with_systemd:%{_unitdir}/netdata.service}
+%{_sysconfdir}/logrotate.d/%{name}
 %{_libexecdir}/%{name}
 %{_sbindir}/%{name}
 %dir %{_datadir}/%{name}
+%dir %{_sharedstatedir}/%{name}
```

Comment:
This fixes this: https://github.com/firehol/netdata/pull/728#issuecomment-235740440